Test/name with space/run.dfy.expect

```diff
@@ -0,0 +1,3 @@
+
+Dafny program verifier finished with 0 verified, 0 errors
```

Comment:
This file was reproducing the previous error. I found that only ".LocalPath" did not have %20 in the file name.